reactkarma/reactkarma.py

```diff
@@ -191,13 +191,24 @@ async def _check_reaction(
         emoji = reaction.emoji
         upvote = await self._is_upvote(guild, emoji)
         if upvote is not None:
-            await self._add_karma(author, 1 if upvote == added else -1)
+            await self._add_karma(user, author, 1 if upvote == added else -1, reaction)
```

Comment:
nitpick: even though ternaries are easy to write, sometimes they're hard to read and reason about. It's best to use them less often.